### PR TITLE
Add 0.12 for kserve and update branch 0.11 to 0.11.1

### PR DESCRIPTION
**Pull request by Jooho**
Add kserve 0.12.0 ci
Rename kserve 0.11 to 0.11.1 
Update test platform openshift version to 4.15

### CODE REVIEW

**Review by spolti — APPROVED** [review]

**Comment by Jooho** [comment]
/pj-rehearse  ci/rehearse/opendatahub-io/kserve/release-v0.11.1/e2e-fast

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-v0.11.1-e2e-fast

File: ci-operator/config/opendatahub-io/kserve/opendatahub-io-kserve-release-v0.12.0.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
+base_images:
+  golang:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
```
Comment (spolti) [inline ci-operator/config/opendatahub-io/kserve/opendatahub-io-kserve-release-v0.12.0.yaml:5]: should 1.21

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-v0.11.1-e2e-fast

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-v0.11.1-e2e-fast

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-v0.11.1-e2e-fast

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-master-e2e-fast

**Comment by spolti** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-v0.11.1-e2e-fast

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-v0.11.1-e2e-fast

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-v0.11.1-e2e-fast

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-v0.12.0-e2e-fast

**Comment by Jooho** [comment]
Close this and I will create a new pr

**Pull request closed without merging**